src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -340,17 +339,8 @@ private ILocalService GetConfigurationPersister()
 
         private ILocalService GetConfigurationProvider()
```

Comment:
I left `GetConfigurationPersister` method as-is: the ctor of `ConfigurationPersister` contains dependencies that can't be easily new-ed, so I just left the construction here as it was.